workspaces/homepage/plugins/dynamic-home-page/src/utils/constants.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const LEARNING_SECTION_ITEMS = [
+  {
+    title: 'Get started',
+    description: 'Learn about Red Hat Developer Hub.',
+    buttonText: 'Go to Tech Docs',
+    buttonLink:
+      'https://docs.redhat.com/en/documentation/red_hat_developer_hub/',
```

Comment:
@ShiranHi Go to Tech Docs confused me here because its not a "tech docs" its our RHDH Documentation. But its fine if we want stay with that.

Reply:
The idea is to have it go to TechDocs. I've asked our docs folks to produce some limited end-user focused documentation in TechDocs format and this button would take you to it. https://issues.redhat.com/browse/RHDHPAI-602

There's a demo on the summit cluster of how it should really work using these techdocs (not official) https://github.com/benwilcock/rhdh-techdocs/blob/main/rhdh-catalog-info.yaml